lib/rubocop/cop/commissioner.rb

```diff
@@ -7,6 +7,9 @@ module Cop
     class Commissioner
       include RuboCop::AST::Traversal
 
+      RESTRICTED_NODE_TYPES = %i[on_send on_csend].freeze
```

Comment:
Great work 👍 

This constant probably should be named differently, something like `RESTRICTED_CALLBACK_NAMES`.

Reply:
Indeed 😄